### ISSUE TITLE
yolo无法训练

### ISSUE THREAD

**Issue by fvwevewv** [WhitecrowAurora/lora-rescripts#20]
<img width="2559" height="1363" alt="Image" src="https://github.com/user-attachments/assets/466fa26c-3e0c-4cdf-97db-baa76303f1f1" />一直报错

**Comment by WhitecrowAurora** [comment]
控制台发一下,具体参数也发一下

**Comment by WhitecrowAurora** [comment]
beta45中做了一些初步修复,可以尝试一下

**Comment by fvwevewv** [comment]
好的,训练器以能够正常获取训练参数开始训练

**Comment by fvwevewv** [comment]
报错了

**Comment by fvwevewv** [comment]
<img width="1720" height="923" alt="Image" src="https://github.com/user-attachments/assets/9ece03b1-cce0-4527-bebe-2439330d683b" />

**Comment by fvwevewv** [comment]
model_train_type = "yolo"
pretrained_model_name_or_path = "E:/trae_workspace/yolo/yolo11m_eyesbbox.pt"
resume = "E:\\lora2\\Yolo\\yolo11m_eyesbbox.pt"
yolo_data_config_path = "E:\\lora2\\dataset\\yolo_v11m_bbox_mixed\\data.yaml"
train_data_dir = "E:/lora2/dataset/yolo_v11m_bbox_mixed/train"
val_data_dir = "E:\\lora2\\dataset\\yolo_v11m_bbox_mixed\\valid"
class_names = "class0"
output_name = "exp"
output_dir = "E:/lora2/output/yolo"
save_every_n_epochs = 10
epochs = 100
batch = 16
imgsz = 640
workers = 8
device = "0"
seed = 1337
max_train_epochs = 100

**Comment by WhitecrowAurora** [comment]
resume里这个模型需要改一下,不能直接用其他训练器训练出来的中间状态或者旧版训练器训练出来的中间状态检查点,硬要用的话请用weights目录里带matedata的pt文件

**Comment by fvwevewv** [comment]
这个是从官方发布页的预训练权重改了个名字,也是不能用的吗

**Comment by WhitecrowAurora** [comment]
已完成训练的yolo模型无法续训,里面不带optimizer,AMP scaler,epoch之类的训练状态信息,只有训练好的权重

**Comment by WhitecrowAurora** [comment]
如果是想微调官方练好的yolo模型,去掉resume里的模型继续训练就可以了

**Comment by fvwevewv** [comment]
好的,谢谢,我去试一下
